frontend/scripts/metrica.js

```diff
@@ -51,16 +51,31 @@
                 metricaConfig[1] = [];
                 metricaConfig[1].push('tracking_id', this.getTrackingIdFromUrl());
             }
+
+            if(metricaConfig.length === 2 || metricaConfig[2].length < 2) {
+                metricaConfig[2] = [];
+                metricaConfig[2].push('spa', this.getSPAFromUrl());
+            }
+
             this.configMetrica = metricaConfig;
         },
+        getScriptSRC() {
+            return document.querySelector(`script[src^='${state.host}metrica.js?']`).src;
+        },
         getTrackingIdFromUrl(){
-            let myScript = document.querySelector(`script[src^='${state.host}metrica.js?']`);
-            let result = (myScript.src || '').match(/tracking_id=(\d+)/i);
+            let result = (this.getScriptSRC() || '').match(/tracking_id=(\d+)/i);
             if (result === null)
                 throw new Error('Tracking id not found');
 
             return result[1];
         },
+        getSPAFromUrl(){
+            let result = (this.getScriptSRC() || '').match(/spa=(\d+)/i);
```

Comment:
this regular expression won't retrieve neither "true" nor "false".
for boolean you can do like this:

return /spa=true/i.test(this.getScriptSRC() || '');